status/versions.go

```diff
@@ -223,6 +224,16 @@ func parseIstioRawVersion(rawVersion string) (*ExternalServiceInfo, error) {
 			}
 			// we know this is Istio upstream - either a supported or unsupported version - return now
 			return &product, nil
+		} else if strings.Contains(istioVersionStringArr[0], "dev") && len(istioVersionStringArr) > 2 {
```

Comment:
because you are going to be accessing the [3] and [4] indexes down on lines 229/230, you need to check that the length is good - otherwise you will throw an array index out of bounds error here if the regex is given something we are not expecting.
```suggestion
		} else if strings.Contains(istioVersionStringArr[0], "dev") && len(istioVersionStringArr) > 4 {
```

Reply:
@jmazzitelli Done.